tables/akk-sumero.uti

```diff
@@ -0,0 +1,966 @@
+
+# liblouis: Sumero-Akkadian
+#
+# ------------
+#-index-name: Sumero-Akkadian uncontracted
+#-display-name: Sumero-Akkadian uncontracted
```

Comment:
I don't really understand where the "Sumero" in the name of this table comes from. If I understand correctly, the difference between this table and the Akkadian one is in the sign inventory the table is based on. But "Cuneiform", "Akkadian" and "Sumero-Akkadian" seem to be used as synonyms, and I can't find any relation between these different names for the script and the different sign inventories that exist. So I'm wondering if these are the best choices for the table names?

By the way, but what exactly is meant by the "American" system? I've looked at the [WIkipedia link](https://en.wikipedia.org/wiki/Cuneiform_(Unicode_block)) you referred to but it's not totally clear.

Reply:
@yeatersink I see you change the `#+display-name` of the table in the last revision. But what is the explanation? "Sumero" is still in other metadata. I'd like to understand what is meant by it and how it related to the sign inventory.

We will somehow need to explain the difference between the two tables in the display name. The tables can both have the same `#+language` metadata, but depending on the software users may decide which one to pick based on the display name.